### PR TITLE
vendor containers/image for parallel copying of layers 

### PR DESCRIPTION
**Pull request by vrothberg**
Vendor the latest containers/image 50e5e55e46a391df8fce1291b2337f1af879b822
to enable parallel copying of layers.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@runcom @vrothberg Lets cut  a new release

**Comment by vrothberg** [comment]
SGTM. I am about to perform some benchmarks comparing pull-performance of Docker, old Skopeo and new Skopeo.

**Comment by runcom** [comment]
I'm tagging a new skopeo now.

**Comment by rhatdan** [comment]
:+1:

File: vendor/github.com/containers/image/docs/atomic-signature.md [hunk 1]
```diff
@@ -1,241 +0,0 @@
-% atomic-signature(5) Atomic signature format
```
Comment (mtrmac) [inline vendor/github.com/containers/image/docs/atomic-signature.md]: In the future, please use `make vendor` instead of manual `vndr` runs.

**Pull request merged**